src/components/text-field-search.js

```diff
@@ -94,6 +96,32 @@ export default class TextFieldSearch extends React.Component {
         )
     }
 
+    handleFilter = (event, clearSelection) => {
+        this.props.clearFields()
+        if (event.target.value === '') {
+            clearSelection()
+            this.suggestions = []
+            this.setState({
+                suggestionsList: [],
+            })
```

Comment:
Are you sure `clearSelection()` is needed? I removed it and the list still cleared as expected. Also, what are the different purposes of `suggestions` and `suggestionsList` ?

Reply:
Hi Jennifer! `clearSelection()` is a downshift action that clears the selection by cleaning some internal states like: 

```
selectedItem: null,
inputValue: ''
```

And suggestions and suggestionsList are basically the same but before the last change this.state.suggestionsList was not updating as it was supposed, I check it again and it's working so I'm fixing the duplicated state/property.